docs/docsite/rst/reference_appendices/general_precedence.rst

```diff
@@ -167,7 +167,7 @@ This category only applies to things that take direct options, generally modules
 
 Outside of task actions, the most recognizable 'direct assignments' are with lookup, filter and test plugins:
 
-.. code::
+.. code:: jinja
```

Comment:
`jinja` is for Jinja templates. If the following would be wrapped in `{{ }}` it would be the right lexer, but for pure Jinja code it is not.

`python` would work *mostly*, except for the `|`.

Maybe keep `text` here? Or add `{{ }}` around every non-empty line?